gradle/libs.versions.toml

```diff
@@ -1,24 +1,26 @@
 [versions]
 compileSdkVersion = "31"
-exampleCompileSdk = "33"
+exampleCompileSdk = "34"
 minSdk = "24"
 java = "1.8"
 buildTools = "30.0.3"
 
 androidGradlePlugin = "7.4.2"
-kotlin = "1.7.20"
-kotlinx_coroutines = "1.7.1"
-kotlinx_serialization = "1.5.1"
-maven_publish_plugin = "0.17.0"
 androidx_lifecycle = "2.6.1"
 app_compat = "1.6.1"
 core_ktx = "1.10.1"
 junit = "4.13.2"
+kotlin = "1.9.0"
```

Comment:
mostly re-ordering

Reply:
Sent you some comments regarding this offline

Reply:
decided to revert for now.